zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/TenantCreateUpdateHandler.java

```diff
@@ -56,7 +56,10 @@ public TenantEntity createNewEntity(final String id) {
   @Override
   public void updateEntity(final Record<TenantRecordValue> record, final TenantEntity entity) {
     final TenantRecordValue value = record.getValue();
-    entity.setKey(value.getTenantKey()).setTenantId(value.getTenantId()).setName(value.getName());
+    entity
+        .setTenantKey(value.getTenantKey())
+        .setTenantId(value.getTenantId())
+        .setName(value.getName());
```

Comment:
❌ We need to add a "parent" join relation here, do define this as the "parent" entity:

```suggestion
    final var joinRelation = TenantIndex.JOIN_RELATION_FACTORY.createParent();
    entity
        .setTenantKey(value.getTenantKey())
        .setTenantId(value.getTenantId())
        .setName(value.getName())
        .setJoin(joinRelation);
```